lib/Dialect/ESI/runtime/cosim/cosim_dpi_server/DpiEntryPoints.cpp

```diff
@@ -61,7 +62,8 @@ static void log(char *epId, bool toClient,
 static int findPort() {
   const char *portEnv = getenv("COSIM_PORT");
   if (portEnv == nullptr) {
-    printf("[COSIM] RPC server port not found. Letting CapnpRPC select one\n");
+    printf(
```

Comment:
I use it here only because the rest of the file uses printf. Originally, it was legacy coming from code which Lenharth wrote while still at Microsoft. I stuck with it because avoiding including <iostream> is a good thing. Since then, I've given in.